content/docs/5.1/concepts/explore-tikv-features/replication-and-rebalancing.md

```diff
@@ -67,36 +66,43 @@ To view the Grafana: http://127.0.0.1:3000
 
 ## Step 2: Write data
 
-On another terminal session, we will use [go-ycsb](https://github.com/pingcap/go-ycsb) to launch a workload.
+On another terminal session, you can use [go-ycsb](https://github.com/pingcap/go-ycsb) to launch a workload.
 
 1. Clone the `go-ycsb` from GitHub.
-    ```sh
+
+    ```shell
```

Comment:
Using shell will cancel the highlight in editor like `vscode`

Reply:
Because users only need to see the final output, not the sources in editors, so do we need to apply this comment? @marsishandsome 
https://deploy-preview-251--tikv.netlify.app/docs/5.1/concepts/explore-tikv-features/replication-and-rebalancing/

Reply:
I cannot see any difference between 
- https://deploy-preview-251--tikv.netlify.app/docs/5.1/concepts/explore-tikv-features/replication-and-rebalancing/
- https://tikv.org/docs/5.1/concepts/explore-tikv-features/replication-and-rebalancing/